packages/x-charts/src/ScatterChart/ScatterChart.test.tsx

```diff
@@ -69,13 +69,43 @@ describe('<ScatterChart />', () => {
       </div>,
     );
     const svg = document.querySelector<HTMLElement>('svg')!;
-    const marks = document.querySelectorAll<HTMLElement>('circle');
 
-    fireEvent.pointerEnter(marks[0]);
+    fireEvent.pointerEnter(svg); // Trigger the tooltip
+    fireEvent.pointerMove(svg, {
+      clientX: 10,
+      clientY: 10,
+    }); // Set tooltip position voronoi value
```

Comment:
Odd that this was necessary 🤔 

I suppose it is because it is using `fireEvent` instead of `userEvent`, which would handle moving/hover/etc